### ISSUE TITLE
configure -D yields floating divide by zero on write

### ISSUE THREAD

**Issue by davegill** [wrf-model/WRF#688]
Taken from an email:

@stacywalters @pedro-jm 

I have been trying to run the official WRF4.0.1 release compiled with configure -D but it is not working. WRF does not run, but, actually I am not able to run Real with configure -D. I compiled with intel and MPI (option 15). I tried to turn off different options in the namelist but Real does not run. I get a "floating divide by zero" error when Real is writing.

If I compile without the -D option in the configure WRF runs fine.

Having the configure -D working is very useful for model development. 
If you any idea of what could be happening please let me know.

ps, I you need a case showing the problem you can find it here:
/glade/scratch/jimenez/wrfout/wrfsolar-v2/d02/2018041615/run

**Comment by smileMchen** [comment]
@davegill 
Yes wrf.exe cannot run when compiled with ./configure -D. I found this when I tried to find the where the rdlai= true case failed.
However, real.exe runs fine.
